src/app-layout/__integ__/app-layout-toolbar-tooltips.test.ts

```diff
@@ -121,6 +120,42 @@ describe.each(['visual-refresh', 'classic'] as const)('%s', theme => {
         await expect(page.isExisting(`.${tooltipStyles.root}`)).resolves.toBe(false);
       })
     );
+
+    describe.each(['bottom', 'side'] as const)('splitPanelPosition=%s', splitPanelPosition => {
+      test(
+        'No split panel trigger tooltip showing for mouse interactions',
+        setupTest({ size, theme, splitPanelPosition }, async page => {
+          await expect(page.getElementsCount(`.${tooltipStyles.root}`)).resolves.toBe(0);
+          await page.hoverElement(wrapper.findSplitPanelOpenButton().toSelector());
+          await expect(page.isExisting(`.${tooltipStyles.root}`)).resolves.toBe(false);
```

Comment:
I don't quite get it. Why the tooltip is not shown on hover here? Because it should be shown only for visual-refresh-toolbar theme?

Reply:
this will be removed once tooltips roll out to visual refresh. This is just to assert they do not exist where they shouldn't